lib/models/second_survey/view/second_question_widget.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:kaizen/models/second_survey/bloc/second_survey_bloc.dart';
+import '../question.dart';
+
+class SecondQuestionWidget extends StatelessWidget {
+  final String mapKey;
+  final Question question;
+
+  const SecondQuestionWidget(
+      {super.key, required this.mapKey, required this.question});
```

Comment:
```suggestion
class SecondQuestionWidget extends StatelessWidget {
  final Question question;
  final void Function(double)? onChanged;

  const SecondQuestionWidget(
      {super.key, required this.question, required this.onChanged});
```

Questo è il modo per non avere l'index dentro la `Question`. In questo modo il widget diventa agnostico a Bloc e non si aggiungono informazioni non "necessarie" nel modello.

Se lasci così va bene lo stesso, giusto per speigarti che intendevo oggi.

Reply:
Ok effettivamente non ci avevo pensato. Preferisco il tuo approccio